HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt

```diff
@@ -0,0 +1,239 @@
+            ) {
+                addToLogView("$placementName(HeliumRewardedAd) didReceiveWinningBid")
+                addToLogView(bidInfo.toString())
+            }
+
+            override fun didCache(placementName: String, error: HeliumAdError?) {
+                if (error != null) {
+                    addToLogView(placementName + "(HeliumRewardedAd) didCache failed with error: " + error.getMessage())
+                } else {
+                    Handler(Looper.getMainLooper()).post {
+                        binding.btnShowRewarded.isEnabled = true
+                    }
+                    addToLogView("$placementName(HeliumRewardedAd) didCache")
+                }
+            }
+
+            override fun didShow(placementName: String, error: HeliumAdError?) {
+                if (error != null) {
+                    addToLogView(placementName + "(HeliumRewardedAd) didShow failed with error: " + error.getMessage())
+                } else {
+                    addToLogView("$placementName(HeliumRewardedAd) didShow")
+                }
+            }
+
+            override fun didClose(placementName: String, error: HeliumAdError?) {
+                if (error != null) {
+                    addToLogView(placementName + "(HeliumRewardedAd) didClose failed with error: " + error.getMessage())
+                } else {
+                    addToLogView("$placementName(HeliumRewardedAd) didClose")
+                }
+            }
+
+            override fun didReceiveReward(placementName: String, reward: String) {
+                addToLogView("$placementName(HeliumRewardedAd) didReceiveReward: $reward")
+            }
+        })
+        return rewardedAd
+    }
+
+    private fun setupScreen() {
+        binding.btnShow.isEnabled = false
+        binding.btnShowRewarded.isEnabled = false
+        binding.interstitialPlacementName.text = getString(R.string.heliumInterstitial)
+        binding.rewardedPlacementName.text = getString(R.string.heliumRewarded)
+    }
+
+    private fun setupLogView() {
+        binding.logView.movementMethod = ScrollingMovementMethod.getInstance()
+        binding.logView.isFocusable = true
+        binding.logView.isFocusableInTouchMode = true
+    }
+
+    private fun addToLogView(s: String) {
+        Handler(Looper.getMainLooper()).post {
+            binding.logView.append(
+                """$s""".trimIndent()
+            )
+        }
+    }
+
+    private fun setupListeners() {
+        binding.btnClean.setOnClickListener { binding.logView.text = "" }
+    }
+
+    private fun setupSdk() {
+        HeliumSdk.setSubjectToCoppa(false)
+        HeliumSdk.setSubjectToGDPR(false)
+        HeliumSdk.setUserHasGivenConsent(false)
+        HeliumSdk.setCCPAConsent(true)
+        addToLogView("Helium SDK Started successfully")
+    }
+
+    override fun onBackPressed() {
+        // If an interstitial is on screen, close it.
+        if (HeliumSdk.onBackPressed()) return else super.onBackPressed()
```

Comment:
Don't forget about the small fix we made to stop crashes on a few devices in a previous pull request. Check the Java side currently and see. It's kinda hard to describe. Feel free to just copy it over. It should auto convert.

Reply:
Thank you Jose, I was not aware (or I didn't remember) this change